src/math/Quaternion.cpp

```diff
@@ -154,4 +137,11 @@ std::ostream& operator<<(std::ostream& os, const Quaternion& q) {
     return os;
 }
 
+scrimmage::Quaternion operator*(const scrimmage::Quaternion &p, const scrimmage::Quaternion &q) {
+    Quaternion pq;
```

Comment:
I don't think pq is used. Right?

Reply:
let's see if I can manage to stop screwing this up